arlo_server/routes.py

```diff
@@ -1255,16 +1258,9 @@ def logout():
 
     clear_loggedin_user()
 
-    # request auth0 logout and come back here when that's done
-    return_url = f"{request.host_url}"
-    params = urllib.parse.urlencode({"returnTo": return_url})
-
-    base_url = (
-        AUDITADMIN_AUTH0_BASE_URL
-        if user_type == UserType.AUDIT_ADMIN
-        else JURISDICTIONADMIN_AUTH0_BASE_URL
-    )
-    return redirect(f"{base_url}/v2/logout?{params}")
+    # because we have max_age on the oauth requests now,
+    # we don't need to log out of Auth0 anymore.
```

Comment:
This is the nittiest nitpick ever, but can you take out "now" and "anymore" from this comment? I have a (surprisingly) strong opinion that comments should be timeless since future readers won't know what came before

Reply:
I want to frame this comment. ;)